magellan-library/src/main/java/com/wealthfront/magellan/navigation/LinearNavigator.kt

```diff
@@ -20,7 +20,7 @@ import com.wealthfront.magellan.transitions.DefaultTransition
 import com.wealthfront.magellan.view.whenMeasured
 import java.util.Stack
 
-class LinearNavigator(
+class LinearNavigator internal constructor(
```

Comment:
This makes the "Single navigator" check a compile time thing so we don't need any lint checks for this.

Reply:
It also makes it so that others can't build their custom journeys with a linear navigator, right? That might not be what we want

Reply:
Yeah. That was my assumptions. You could always build your own journey if you want that.